README.md

```diff
@@ -16,6 +16,7 @@ Each subdirectory contains relevant code and instructions for the corresponding
 - [Clean and Standardize Data](clean-and-standardize-data): Sample code for cleaning and standardizing data.
 - [Data Mapper](data-mapper): Sample code for data mapping between different structures.
 - [Enrich Data](enrich-data): Sample code to enrich data.
+- [Sentiment Analysis](sentiment-analysis): Sample code for analyse feedbacks.
```

Comment:
Shall we remove the earlier one?

Reply:
removed from the README but didn't remove the sample.